port: /dev/ttyDXL_leader_right

groups:
  arm:
    torque_enable: false

singles:
  gripper:
    torque_enable: false
